app/views/templates/project.html.slim

```diff
@@ -0,0 +1,16 @@
+.container-fluid
+  .page-header
+    h1 {{ project | projectName }}
+  .row.overview
+    .col-lg-3.col-md-4.col-xs-12
+      .summary.panel.panel-default ng-show='project.id'
+        .panel-heading.clearfix
+          .label.label-info.pull-right tooltip='Project ID' {{ project.id }}
```

Comment:
Add the `.apiId` class to this label so that the ID is displayed with a Monospace font.